src/components/main.jsx

```diff
@@ -122,6 +127,22 @@ export default class Main extends React.Component {
     if (this.state.tool === 'measure') {
       this.setState({ lengthMeasurePoints: [ ...this.state.lengthMeasurePoints, { lat, lon } ] });
     }
+    if(this.state.tool == 'route-planner'){
```

Comment:
no ak to nebudes robit, tak to budem pridavat za teba a vezme mi to kus casu... ale mozem sa skusit pohrat s automatickym fixovanim pomocou eslintu